README.md

```diff
@@ -344,9 +344,16 @@ Volume resource for HPE OneView.
 ```ruby
 oneview_volume 'Volume_1' do
   client <my_client>
+  snapshot_data <snapshot_data>
+  action [:create_snapshot, :delete_snapshot]
+end
+```
+
+```ruby
+oneview_volume 'Volume_1' do
+  client <my_client>
   data <resource_data>
-  storage_system_name <storage_system_name>
-  storage_system_ip <storage_system_ip>
+  storage_system <storage_system>
```

Comment:
Can you update the notes below too? They still show storage_system_name and storage_system_ip.